src/anaconda/NOTES.md

```diff
@@ -0,0 +1,13 @@
+## Using Conda
+
+This Feature includes [the `conda` package manager](https://aka.ms/vscode-remote/conda/about). Additional packages installed using Conda will be downloaded from Anaconda or another repository if you configure one. To reconfigure Conda in this container to access an alternative repository, please see information on [configuring Conda channels here](https://aka.ms/vscode-remote/conda/channel-setup).
```

Comment:
nit: Should we update the links to be https://docs.conda.io/projects/conda/en/latest/index.html and https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html directly, rather than the aka.ms links, to be a bit more agnostic in naming?

Reply:
Yeah, you are right - no reason to go through a Microsoft redirector. Will update.